nuvlaedge/agent/orchestrator/kubernetes.py

```diff
@@ -86,6 +89,25 @@ def _sanitize_k8s_object(cls, data):
         else:
             return data
 
+    @staticmethod
+    def _list_helm_releases():
+        cmd = 'helm list --all-namespaces -o json'
+        try:
+            result = run(cmd, capture_output=True,
+                         timeout=10, encoding='UTF-8',
+                         shell=True, check=False)
+            if result.returncode != 0:
+                log.error('Failed to list Helm releases: %s', result.stderr)
+                return []
+            log.debug('Helm list command output: %s', result.stdout)
+            return json.loads(result.stdout)
+        except json.JSONDecodeError as e:
+            log.error('Failed to parse Helm list output: %s, %s', e, result.stdout)
+            return []
+        except TimeoutExpired:
+            log.warning('Timed out waiting for Helm list command.')
+            return []
+
```

Comment:
I would add a single return at the end.
```suggestion
        except json.JSONDecodeError as e:
            log.error('Failed to parse Helm list output: %s, %s', e, result.stdout)
        except TimeoutExpired:
            log.warning('Timed out waiting for Helm list command.')
        return []

```

Reply:
Yes. Accepted.